### ISSUE TITLE
Funky output when a package is missing

### ISSUE THREAD

**Issue by rtfeldman** [dnalot/elm-sublime#5]
To reproduce:

1. Add a new dependency to `elm-package.json` but do not install it yet
2. Attempt a build in Sublime

Here's what I got:

![screen shot 2015-08-07 at 12 00 18 am](https://cloud.githubusercontent.com/assets/1094080/9130413/55c58284-3c97-11e5-8d2a-d801475a23e4.png)

**Comment by dnalot** [comment]
I can definitely fix this. It's caused by elm-lang/elm-make#38

**Comment by dnalot** [comment]
The broken lines are fixed (ca7ba1c) in my branch:
```mediawiki
=== Error reading file elm-package.json: ===
=== Dependency file has an invalid name: elm-html ===
=== Must have format USER/PROJECT and match a public github project. ===
```
Strings were being chunked into output instead of buffered. Strangely only for this kind of error message. What I don't know how to fix is parsing these lines to be formatted as a single error instead of three independent lines. Currently I parse by:

0. [`split`](https://github.com/dnalot/elm-sublime/blob/ca7ba1cc3c3a9b1f8a95a51a927c7017ae597217/elm_make.py#L67) `elm-make` output on newlines
0. [`try`](https://github.com/dnalot/elm-sublime/blob/ca7ba1cc3c3a9b1f8a95a51a927c7017ae597217/elm_make.py#L75-L77) to decode JSON
0. [`format`](https://github.com/dnalot/elm-sublime/blob/ca7ba1cc3c3a9b1f8a95a51a927c7017ae597217/elm_make.py#L83) as plain text if invalid

I considered concatenating sequential non-JSON "`info`" messages but they're not necessarily related. @evancz Can you recommend a workaround? The code above motivated elm-lang/elm-make#38.
